consensus-types/types/payload.go

```diff
@@ -561,14 +561,11 @@ func (p *ExecutionPayload) GetExcessBlobGas() math.U64 {
 }
 
 // ToHeader converts the ExecutionPayload to an ExecutionPayloadHeader.
-func (p *ExecutionPayload) ToHeader() (
-	*ExecutionPayloadHeader,
-	error,
-) {
+func (p *ExecutionPayload) ToHeader() (*ExecutionPayloadHeader, error) {
 	txsRoot := p.GetTransactions().HashTreeRoot()
 
 	switch p.Version() {
-	case version.Deneb, version.DenebPlus:
```

Comment:
@calbera again idea here is that we have never used Deneb1 so this can go?

Reply:
correct